tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -403,6 +403,10 @@ build_rootfs_distro()
 		before_starting_container
 		trap after_stopping_container EXIT
 
+		[ -z "${AGENT_SOURCE_BIN} " ] && die "need agent binary"
```

Comment:
```suggestion
		[ -z "${AGENT_SOURCE_BIN}" ] && die "need agent binary"
```

or it will never be true